demo/opae-nlb-demo/Dockerfile

```diff
@@ -1,24 +1,13 @@
-# CLEAR_LINUX_BASE and CLEAR_LINUX_VERSION can be used to make the build
-# reproducible by choosing an image by its hash and installing an OS version
-# with --version=:
-# CLEAR_LINUX_BASE=clearlinux@sha256:b8e5d3b2576eb6d868f8d52e401f678c873264d349e469637f98ee2adf7b33d4
-# CLEAR_LINUX_VERSION="--version=29970"
-#
-# This is used on release branches before tagging a stable version.
-# The main branch defaults to using the latest Clear Linux.
-ARG CLEAR_LINUX_BASE=clearlinux:latest
+# Fedora is a recommended distro for OPAE
+# https://opae.github.io/latest
+ARG FEDORA_RELEASE="32"
+FROM fedora:${FEDORA_RELEASE} as builder
 
-FROM ${CLEAR_LINUX_BASE} as builder
-
-ARG CLEAR_LINUX_VERSION=
-
-RUN swupd update --no-boot-update ${CLEAR_LINUX_VERSION} && \
-    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc devpkg-tbb git
 # Fetch dependencies and source code
-ARG OPAE_RELEASE=1.5.0-2
+ARG OPAE_RELEASE=2.0.2-1
+
+RUN dnf install -y git-core wget cmake gcc gcc-c++ make spdlog-devel kernel-headers libedit-devel libuuid-devel json-c-devel cli11-devel python-devel
 
-# workaround for a swupd failure discussed in https://github.com/clearlinux/distribution/issues/831
-RUN ldconfig
 RUN mkdir -p /usr/src/opae && \
     cd /usr/src/opae && \
     wget -q https://github.com/OPAE/opae-sdk/archive/${OPAE_RELEASE}.tar.gz -O- | tar -zx
```

Comment:
as part of #643 we've started adding checksum tests. we could do the same here too.

See: https://github.com/intel/intel-device-plugins-for-kubernetes/pull/680/files#diff-ea2f7f21793ed2272669aac75eb4b70d44812963b362646fc63baaefea58f7d8R26-R27

Reply:
done